config/pintool.mpb

```diff
@@ -14,11 +14,11 @@ project : pin {
   verbatim (make, macros) {
     ifeq ($(shell uname -s), Darwin)
       CPPFLAGS += -DTARGET_MAC
-      LDFLAGS  += -lpindwarf -lpinpthread -Wl,-exported_symbols_list -Wl,$(PIN_ROOT)/source/include/pin/pintool.exp
+      LDFLAGS  += $(PIN_ROOT)/intel64/runtime/pincrt/crtbeginS.o -nostdlib -ldl-dynamic -lunwind-dynamic -lpin3dwarf -lpinpthread -Wl,-exported_symbols_list -Wl,$(PIN_ROOT)/source/include/pin/pintool.exp
       LDFLAGS  -= -lpthread
     else
-      CPPFLAGS += -DTARGET_LINUX -Wl,--hash-style=sysv
-      LDLIBS   += -lpindwarf
+      CPPFLAGS += -DTARGET_LINUX -fno-rtti -Wl,--hash-style=sysv $(PIN_ROOT)/intel64/runtime/pincrt/crtendS.o
+      LDLIBS   += -nostdlib -ldl-dynamic -lunwind-dynamic -lpin3dwarf -L$(PIN_ROOT)/intel64/runtime/pincrt
```

Comment:
Same here, doesn't look correct to have intel64 in a generic part

Reply:
moved out of generic part.